src/effects/effectchain.cpp

```diff
@@ -290,17 +300,14 @@ EffectChainPointer EffectChain::createFromXml(EffectsManager* pEffectsManager,
     QString insertionTypeStr = XmlParse::selectNodeQString(element,
                                                            EffectXml::ChainInsertionType);
 
-    EffectChain* pChain = new EffectChain(pEffectsManager, id);
+    EffectChainPointer pChain = EffectChainPointer(new EffectChain(pEffectsManager, id));
```

Comment:
Please omit the assignment + redundant constructor invocation here:
```
EffectChainPointer pChain(new EffectChain(pEffectsManager, id));
```
I prefer the more intuitive assignment version nowadays when using one of the std:make_something() variants. The assignment actually will be removed by the compiler:
```
EffectChainPointer pChain = std::make_shared<EffectChain>(pEffectsManager, id);
```
But EffectChainPointer is still a QSharedPointer ;)